packages/api/src/services/actions.ts

```diff
@@ -22,6 +25,23 @@ export function Handlers(config: Json, libs: Libs): Actions {
       const blocks = await libs.blocks.values();
       return blocks.slice(start, end);
     },
+    async collateralAddresses() {
+      return Array.from(libs.collateralAddresses.values());
+    },
+    async syntheticAddresses() {
+      return Array.from(libs.syntheticAddresses.values());
+    },
+    async allLatestPrices(currency: "usd" = "usd") {
+      assert(exists(libs.prices[currency]), "invalid currency type: " + currency);
+      return libs.prices[currency].latest;
+    },
+    async latestPriceByAddress(address: string, currency: "usd" = "usd") {
```

Comment:
what is this "usd" = "usd" syntax?

Reply:
its meant to say currency can only be of type "usd", and if nots provided, then set it to "usd" as a default. I probably should break this out into its own type, i just anticipate we may want to denominate prices in other units, potentially like "eth", or "btc" at some point.

I'll move this into its own type, since i do this in a few places